ChiModules/LinearBoltzmannSolver/lbs_vectorassembly.cc

```diff
@@ -36,28 +23,28 @@ AssembleVector(LBSGroupset& groupset, Vec x, double *y,
     {
       for (int m=0; m<num_moments; m++)
       {
-        int mapping = transport_view.MapDOF(i,m,gsi);
-        double* source_mapped = &y[mapping];
+        size_t mapping = transport_view.MapDOF(i,m,gsi);
         for (int g=0; g<gss; g++)
         {
           index++;
-          x_ref[index] = source_mapped[g]; //On purpose
+          x_ref[index] = y[mapping+g]; //On purpose
```

Comment:
I do not understand the on purpose comment. Can you elaborate on the comment for futur code developers? Thanks!

Reply:
Done. The comment is now `//Offset on purpose`.